mlflow/experiments.py

```diff
@@ -38,7 +38,31 @@ def list_experiments():
     List all experiments in the configured tracking server.
     """
     store = _get_store()
-    experiments = store.list_experiments()
+    experiments = store.list_experiments("deleted_only")
     table = [[exp.experiment_id, exp.name, exp.artifact_location if is_uri(exp.artifact_location)
               else os.path.abspath(exp.artifact_location)] for exp in experiments]
     print(tabulate(sorted(table), headers=["Experiment Id", "Name", "Artifact Location"]))
+
+
+@commands.command("delete")
+@click.argument("experiment_id")
+def delete_experiment(experiment_id):
+    """
+    Delete an experiment from backend store.
+    Will error out if experiment does not exist or already deleted.
```

Comment:
This help should say that you mark it for deletion. Experiments marked this way can be restored with `restore`, or cleared based on the backend store (refer to docs for that).

Reply:
done